library/tract-renderer/src/main/java/org/cru/godtools/tract/model/Manifest.java

```diff
@@ -114,6 +114,11 @@ public String getCode() {
     }
 
     @NonNull
+    public String getCodeForShareActivity() {
+        return mCode.equals("kgp-us") ? "kgp" : mCode;
```

Comment:
I think I want to keep this custom sharing override logic localized to the TractActivity for now.